src/DataCollector/GuzzleDataCollector.php

```diff
@@ -42,7 +42,7 @@ public function __construct(
     /**
      * {@inheritdoc}
      */
-    public function collect(Request $request, Response $response, \Exception $exception = null)
+    public function collect(Request $request, Response $response, ?Throwable $exception = null)
```

Comment:
this change would break backward compatibility with previous versions of PHP

Reply:
Since what version of PHP does this work?
Should I require higher version of PHP?

Reply:
nullable types was added in https://www.php.net/releases/7_1_0.php while this bundle should work with older versions of PHP (">=5.6)

Reply:
Do you propose any solution?

Reply:
I propouse to leave this line unchanged in order to keep PHP 5.6 support